.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test
```

Comment:
Can you update this to something like `Run Crawlers`?

Reply:
ok sure